hack/lib/serverless.bash

```diff
@@ -244,7 +244,8 @@ metadata:
 spec:
   broker:
     enabled: true
-    bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
+    defaultConfig:
+      bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
```

Comment:
This seems to be a breaking change. How are customers supposed to upgrade? 🤔

Reply:
Nope, b/c this was introduced a week ago or so, was not there on 1.19 (the `broker` things are new)

Reply:
Ahhh, nevermind then, thanks!